eng/Singing.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup Condition="'$(OfficialBuild)' != 'true'">
+  	<!-- Remove symbol nuget packages from items to sign when not in an official build -->
+    <ItemsToSign Remove="%(ItemsToSign.Identity)" Condition="$([System.String]::Copy('%(ItemsToSign.Identity)')).EndsWith('.symbols.nupkg')" />
```

Comment:
I think with the newer versions of MSBuild you don't need to do the `::Copy` thing to be able to call an EndsWith, but I'm not sure :-)

Reply:
That might be true, but we need to work with the MSBuild that comes with VS2017/VS2019 too, so if it doesn't work there I will have to keep it as is.